src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -201,6 +201,98 @@ After a short (less than 5 minute) delay, the status will change from **Pending
 
 </section>
 
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+## Step 5. (Optional) Configure private endpoint trusted owners
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/limited-access.md %}
+{{site.data.alerts.end}}
+
+Optionally, you can restrict the AWS accounts that can connect to your cluster privately. During Limited Access, to configure trusted owners, you must use the [CockroachDB CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}) or [Terraform Provider](https://github.com/cockroachdb/terraform-provider-cockroach/).
+
+Keep the following in mind:
+- This feature can be enabled only on clusters created after your organization is enrolled in the Limited Access.
+- After your organization is enrolled, an attempt to create a PrivateLink connection from another AWS account will fail.
+- After your organization is enrolled, the feature cannot be disabled.
+
+Your service account must have one of the following roles on the cluster, either directly or by inheritance:
+
+- [Cluster Administrator]({% link cockroachcloud/authorization.md %}#cluster-administrator) to add or remove private endpoint trusted owners.
```

Comment:
```suggestion
- [Cluster Administrator]({% link cockroachcloud/authorization.md %}#cluster-administrator) to add, remove, list or get private endpoint trusted owners.
```

Reply:
In the existing API spec, the GET and LIST endpoints only require Cluster Operator. @jaylim-crl can you confirm?